NEXTCLOUD.cmake

```diff
@@ -69,6 +69,16 @@ if(WIN32)
     # MSI Upgrade Code (without brackets)
     set( WIN_MSI_UPGRADE_CODE                "FD2FCCA9-BB8F-4485-8F70-A0621B84A7F4" )
 
+    # CfAPI Shell Extensions
+    set( CFAPI_SHELL_EXTENSIONS_LIB_NAME CfApiShellExtensions )
+    
+    set( CFAPI_SHELLEXT_APPID_REG "{E314A650-DCA4-416E-974E-18EA37C213EA}")
```

Comment:
@mgallien This is up to you. If you want to spin multiple clients simultaneously - then - yes you'd need unique IDs, otherwise, it's going to be a mess. Be advised, that these IDs are not made manually. You'd need Windows GUID tools to generate those during the build. You'd also need to store them somewhere, so, when making a new build of the same custom client you wouldn't want the GUID to ger generated again and again.
If you assume only one Nextcloud client running per one machine - then we don't really have a problem. Each client registers itself at the startup.

Reply:
just to be sure
if I build a specific desktop client with another name (for example NextcloudStaging) may I install both and still have them work in parallel with the same shell extension DLL ?

Reply:
@mgallien This should allow for what you're asking https://github.com/nextcloud/desktop/pull/4713/commits/7b465e20663bd8ad45639f49dcdc8567e7a779af